reana_client/cli/utils.py

```diff
@@ -378,25 +348,17 @@ def output_user_friendly_logs(workflow_logs, steps):
 
             for key, value in logs_info.items():
                 if value:
-                    title = click.style(
-                        "{mark} {description}:".format(
-                            mark=leading_mark,
-                            description=key_to_description_mapping[key],
+                    display_message(
+                        "{description}: {value}".format(
+                            description=key_to_description_mapping[key], value=value,
                         ),
-                        fg=status_to_color_mapping.get(logs_info["status"]),
+                        msg_type=JOB_STATUS_TO_MSG_TYPE.get(logs_info["status"]),
                     )
```

Comment:
m...m... not sure what _same here_ means in this context.

In its original version, this code snippet also considered the _job_status_ to display the message in different colors. If you want to preserve the colors, and avoid the paddings (`==>`), I would need to revert to its original version (using click.secho`).

Reply:
Exactly, that's what I meant (revert back). Sorry if I wasn't explicit enough. We want to avoid all these `SUCCESS/ERROR` messages:
![image](https://user-images.githubusercontent.com/8863238/133633644-e89d28c6-2149-43cf-ad31-ce47ce35ac90.png)

Reply:
Changed in https://github.com/reanahub/reana-client/commit/49de64be82dd4c968ef8edd44d71434c39d06053